### ISSUE TITLE
help with import job in k8s cluster

### ISSUE THREAD

**Issue by andycmaj** [microcks/microcks-cli#15]
hey all.
was hoping someone could help me debug this issue importing from a kubernetes job
```
panic: interface conversion: interface {} is nil, not string

goroutine 1 [running]:
github.com/microcks/microcks-cli/pkg/connectors.(*microcksClient).GetKeycloakURL(0xc0000ae220, 0x0, 0x0, 0x0, 0x0)
	/Users/lbroudou/Development/go-workspace/src/github.com/microcks/microcks-cli/pkg/connectors/microcks_client.go:106 +0x551
github.com/microcks/microcks-cli/cmd.(*importComamnd).Execute(0x966bc8)
	/Users/lbroudou/Development/go-workspace/src/github.com/microcks/microcks-cli/cmd/import.go:82 +0x4df
main.main()
	/Users/lbroudou/Development/go-workspace/src/github.com/microcks/microcks-cli/main.go:31 +0x76
```

command being run looks like this:

```
  - |-
                  microcks-cli import '/schema/test_schema.yaml:true' \
                    --microcksURL=http://microcks-test:8080/api \
                    --keycloakClientId=microcks-serviceaccount \
                    --keycloakClientSecret=****** \
                    --insecure --verbose
```

not sure where to start with this message. it mentions a keycloak url, but that's not a valid cli argument right?

**Comment by lbroudoux** [comment]
Hey !
The keycloak url is in fact retrieved from the Microcks server using the `microcksURL` you provide. Seems like one of those two URLs is not correct. Could you just test using `http://microcks-test:8080/api/` with a trailing `/`?

**Comment by lbroudoux** [comment]
This will be addressed with #18. In the mean time, you should have `http://microcks-test:8080/api/` as your URL.
